pom.xml

```diff
@@ -157,6 +163,19 @@
 			<version>4.1.1</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.junit.platform</groupId>
+			<artifactId>junit-platform-launcher</artifactId>
```

Comment:
Do you really need this dependency? I don't think we run tests outside of Maven or the IDE.

Reply:
We don't need this one; it can be removed.

There is an inconsistency between JUnit's [list of required dependencies](https://junit.org/junit5/docs/current/user-guide/#running-tests-junit-platform-runner-setup), which has 4 required dependencies for running Junit4 and 5 in the same project, and [Surefire's](https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html), which only has two dependencies.

Surefire's seems to work fine from both command line and IDE. *shrug*